manifest.json

```diff
@@ -35,7 +35,8 @@
   ],
   "options_ui": {
     "page": "options.html",
-    "open_in_tab": true
+    "browser_style": true,
```

Comment:
This leads to the following error in Chrome when loading the extension:
found unexpected key 'browser_style'